FirebaseMLModelDownloader/Sources/LocalModelInfo.swift

```diff
@@ -13,17 +13,13 @@
 // limitations under the License.
 
 import Foundation
-import FirebaseCore
 
 /// Model info object with details about downloaded and locally available model.
-// TODO: Can this be backed by user defaults property wrappers?
 class LocalModelInfo {
   /// Model name.
   let name: String
-
```

Comment:
Nit: please leave the space in between here and the others (unless this was done automatically with the formatter... in which case please ignore me 😄 )

Reply:
It might have been the formatter, but I wasn't consistent about this to begin with so I'll add spaces if it makes it more readable.